cli/cmd/cluster_shell.go

```diff
@@ -19,20 +19,23 @@ import (
 
 func (r *runners) InitClusterShell(parent *cobra.Command) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "shell [ID]",
+		Use:   "shell [ID_OR_NAME]",
 		Short: "Open a new shell with kubeconfig configured.",
 		Long: `The 'shell' command opens a new shell session with the kubeconfig configured for the specified test cluster. This allows you to have immediate kubectl access to the cluster within the shell environment.
 
-You can either specify the cluster ID directly or provide the cluster name to resolve the corresponding cluster ID. The shell will inherit your existing environment and add the necessary kubeconfig context for interacting with the Kubernetes cluster.
+You can either specify the cluster ID or name directly as an argument, or provide the cluster name or ID using flags. The shell will inherit your existing environment and add the necessary kubeconfig context for interacting with the Kubernetes cluster.
 
 Once inside the shell, you can use 'kubectl' to interact with the cluster. To exit the shell, press Ctrl-D or type 'exit'. When the shell closes, the kubeconfig will be reset back to your default configuration.`,
-		Example: `# Open a shell for a cluster by ID
-replicated cluster shell CLUSTER_ID
+		Example: `# Open a shell for a cluster by ID or name
+replicated cluster shell CLUSTER_ID_OR_NAME
 
-# Open a shell for a cluster by name
-replicated cluster shell --name "My Cluster"`,
+# Open a shell for a cluster by name using a flag
+replicated cluster shell --name "My Cluster"
```

Comment:
This should be deprecated with the same warning we used for other deprecations in this PR.